Commands/Principals/AddAlert.cs

```diff
@@ -1,37 +1,30 @@
-﻿#if !ONPREMISES
+﻿#if !SP2013 && !SP2016
```

Comment:
@erwinvanhunen Quick note: there are corresponding #ifs around the unit test methods which should be changed as well.

Reply:
Good point. Will fix that for next month's release as we already started the build process for this month.

Reply:
I'll do that and add tests for your Get- and Remove-methods as well.

Reply:
The changes are included in https://github.com/SharePoint/PnP-PowerShell/pull/1965